tests/integration/synapseutils/test_synapseutils_copy.py

```diff
@@ -2,6 +2,7 @@
 import time
```

Comment:
Yes, the copy module tests could be fragile, especially if things are being copied into the same project across parallel tests that are running